### PR TITLE
Added How to Track Pageviews with GTM in a React-Redux app

### PR DESCRIPTION
**Pull request by AmyScript**
Checklist
----

_Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

 - [ ] I have added tests that prove my fix is effective or that my feature works.
 - [ ] I have added all necessary documentation (if appropriate)

What was done
----
Added documentation for GTM pageview tracking example

Associated Issues
----
 - _list any associated issue numbers here_

:heart: Thanks
----
Thanks for taking the time to help out with the project, it's much appreciated :slightly_smiling_face:

### CODE REVIEW

**Comment by ttmarek** [comment]
This is awesome, thanks so much @AmyScript! A couple of things though:

Your example is pretty specific to using GTM with Google Analytics, but you _can_ (and people often do) use GTM with other analytics services.

I think this is a good starting point for something that I've wanted to do for a while. How do you feel about exporting your GTM + GA configuration and we can add it to the repo?

Here's what I'm talking about :point_down: 

image

This is all under the Admin tab in GTM.

If you export container you basically get a `.json` file with your whole configuration. All the Tags, Variables, and Triggers you created. 

Basically what I'm thinking we could do is have a super simple getting started guide for using Redux Beacon with GTM and GA:

 1. Set up Redux Beacon (eventsMap, and eventDefinition)
 2. Import the GTM container (the `.json` file with all Tags, Variables, and Triggers)
 3. Connect GTM to your GA account

And that's it!

I the future we could have configurations for more than just GTM + GA. We can have GTM + whatever:

image


Thoughts?

**Comment by AmyScript** [comment]
@ttmarek sounds good.  Let me get to that next week and revise the PR.
